test/integration/YieldTokenIncidents/submitClaim.js

```diff
@@ -3,8 +3,9 @@ const {
   constants: { ZERO_ADDRESS },
 } = require('@openzeppelin/test-helpers');
 const { expect } = require('chai');
+const { AddressZero } = ethers.constants;
```

Comment:
`AddressZero` was imported but oz is still in use.

Reply:
removed